cmd/delete/key/key.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 IBM Corp
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package key
+
+import (
+	"fmt"
+	"github.com/fatih/color"
+	"regexp"
+
+	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
+
+	"github.com/ppc64le-cloud/pvsadm/pkg"
+	"github.com/ppc64le-cloud/pvsadm/pkg/client"
+	"github.com/ppc64le-cloud/pvsadm/pkg/utils"
+)
+
+var searchStr string
+
+var Cmd = &cobra.Command{
+	Use:   "key",
+	Short: "Delete PowerVS ssh key/keys",
+	Long:  `Delete PowerVS ssh key/keys`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		opt := pkg.Options
+
+		c, err := client.NewClientWithEnv(opt.APIKey, opt.Environment, opt.Debug)
+		if err != nil {
+			klog.Errorf("failed to create a session with IBM cloud: %v", err)
+			return err
+		}
+
+		pvmclient, err := client.NewPVMClientWithEnv(c, opt.InstanceID, opt.InstanceName, opt.Environment)
+		if err != nil {
+			return err
+		}
+
+		keys, err := pvmclient.KeyClient.GetAll(opt.InstanceID)
```

Comment:
see if there is a way of listing keys be created time, to support some of the options we have like `--since`/`--before`, this is mainly to support delete all the keys which are older than so and so days matching to a certain pattern.